src/lib/config.ts

```diff
@@ -54,15 +60,21 @@ async function fetchProfileWearables(profile: Avatar | null, peerUrl: string) {
   if (!profile) {
     return []
   }
-  const results = await fetchURNs(profile.avatar.wearables, peerUrl)
-  return results.filter((result) => !isEmote(result))
+  const [wearables] = await fetchURNs(profile.avatar.wearables, peerUrl)
+  return wearables
 }
 
 async function fetchURNs(urns: string[], peerUrl: string) {
   if (urns.length === 0) {
     return []
   }
-  return peerApi.fetchWearables(urns, peerUrl)
+  const results = await peerApi.fetchItems(urns, peerUrl)
+  return results
```

Comment:
Just a suggestion (the async symbol can also be removed):
```suggestion
  return peerApi.fetchItems(urns, peerUrl)
```